### PR TITLE
accelerometer_observation model and api in place and added to classro…

### PR DESCRIPTION
**Pull request by marknutter**
…om_sim

### CODE REVIEW

**Comment by ps2** [comment]
Could you add a datatype to the api.yaml file that shows the data model that is expected to be uploaded to the AccelerometerObservations api?

File: app/api/accelerometer_observations.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        event_data = request.get_json()
+    if not event_data:
+        abort(400, "Missing event data")
+    if not isinstance(event_data, list):
+        event_data = [event_data]
+
+    classroom_id = event_data[0]['classroom_id']
+
+    mappings = {m.sensor_id: m for m in SensorMapping.query.filter_by(classroom_id=classroom_id, end_time=None)}
+    relationships = {r.key(): r for r in EntityRelationship.query.filter_by(classroom_id=classroom_id)}
```
Comment (ps2) [inline app/api/accelerometer_observations.py]: This doesn't appear to be used.

File: app/models/accelerometer_observation.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import dateutil.parser
+from shared import *
+from sqlalchemy.dialects.postgresql import insert
+from sensor_mapping import MappingType
+
+# Raw radio observation
+class AccelerometerObservation(db.Model):
+    classroom_id = db.Column(db.Integer, nullable=False, primary_key=True)
+    observed_at = db.Column(db.DateTime, nullable=False, primary_key=True)
+    entity_id = db.Column(db.Integer,  db.ForeignKey('entity_relationship.id'), nullable=False, primary_key=True)
```
Comment (ps2) [inline app/models/accelerometer_observation.py]: This foreign key constraint is not correct.  The entity_id referred to here is really an entity in TC, so we can't add a foreign key constraint for it.  Entity *relationships* are a different thing.

**Comment by marknutter** [comment]
@ps2 your comments should be addressed now.

**Pull request merged**